app/helpers/stafftools_resource_render_helper.rb

```diff
@@ -0,0 +1,6 @@
+module StafftoolsResourceRenderHelper
+  def render_stafftools_resource(resource)
+    type = resource._data['_type']
+    render partial: "stafftools/#{type.pluralize}/#{type}", locals: { type.to_sym => resource }
+  end
+end
```

Comment:
Beautiful. Take advantage of the hidden `_type` field and automatically render a partial base on it. Nice job. This is simple and easily scalable.